src/View/Cell.php

```diff
@@ -183,9 +214,7 @@ public function render($template = null)
         };
 
         if ($cache) {
-            return $this->View->cache(function () use ($render) {
-                echo $render();
-            }, $cache);
+            return Cache::remember($cache['key'], $render, $cache['config']);
```

Comment:
It's similar but with an important difference that View::cache() expects the action to have been invoked and it only avoids the rendering of view. The purpose of this patch is to avoid even running of action as requested in the original ticket.

So i need to check for cached data even before invoking action which is not possible by using View::cache()